site/src/pages/WorkspacePage/WorkspaceTopbar.tsx

```diff
@@ -106,6 +106,10 @@ export const WorkspaceTopbar: FC<WorkspaceProps> = ({
     allowAdvancedScheduling,
   );
 
+  const isImmutable =
+    workspace.latest_build.status === "deleted" ||
+    workspace.latest_build.status === "deleting";
```

Comment:
will it also hide the retry action in case of a failed delete?

Reply:
no, the status will switch to `"failed"`